chrome/content/ui.js

```diff
@@ -392,6 +393,14 @@ var ui = {
     Services.ww.openWindow(null, privDialog, "", features, args);
   },
 
+  onAccountCreated: function(acc) {
+    let args = {
+      account: acc.normalizedName,
+      protocol: acc.protocol.normalizedName,
+    };
```

Comment:
You should also check that `otr.privateKeyFingerprint()` returns `null` first, so we don't overwrite a key that already exists.